src/main/webapp/app/exercises/quiz/shared/questions/short-answer-question/short-answer-question.component.html

```diff
@@ -40,8 +40,8 @@ <h4 class="question-title-display">
                         class="short-answer-question-container__input"
                         disabled
                         [style.background]="getBackgroundColourForInputField(element)"
-                        value="{{ getSubmittedTextForSpot(element) !== undefined ? getSubmittedTextForSpot(element).text : '' }}"
-                        size="{{ getSubmittedTextForSpot(element) !== undefined ? getSubmittedTextForSpot(element).text!.length + 2 : 5 }}"
+                        value="{{ getSubmittedTextForSpot(element)?.text !== undefined ? getSubmittedTextForSpot(element).text : '' }}"
```

Comment:
I think you can simplify that line with `getSubmittedTextForSpot(element)?.text ?? ''`

Reply:
@ivanchimeno can you provide an example of how to use the operator ?? in this case? I tried this and it throws a compilation error Expression expected